### PR TITLE
[sd-1950]-(fix + sd-3417)- Links for related item

### PR DESCRIPTION
**Pull request by sjunaid**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

File: client/app/scripts/superdesk-archive/directives.js [hunk 1]
```diff
@@ -245,7 +245,9 @@
                         });
                     });
                     scope.open = function(item) {
-                        superdesk.intent('read_only', 'content_article', item);
+                        superdesk.intent('view', 'item', item).then(null, function() {
+                            superdesk.intent('edit', 'item', item);
+                        });
```
Comment (ioanpocol) [inline client/app/scripts/superdesk-archive/directives.js]: One unittest for this functionality will help.

File: client/app/scripts/superdesk-legal-archive/views/legal_archive.html [hunk 2]
```diff
@@ -66,7 +66,8 @@
                     <div class="filter-trigger" ng-click="openAdvanceSearch = true" ng-class="{hidden: openAdvanceSearch}"><i class="icon-filter-large"></i></div>
                     <div sd-legal-item-sortbar></div>
                     <div sd-pagination data-items="items"></div>
-
+                    <div class="searchbar-flex-handler"></div>
```
Comment (ioanpocol) [inline client/app/scripts/superdesk-legal-archive/views/legal_archive.html]: Seams that this is not related to related items, so at least a different commit is better.

**Comment by sivakuna-aap** [comment]
@sjunaid are you planning to split into 2 commits?
